### ISSUE TITLE
Prototype pollution of dependency yargs

### ISSUE THREAD

**Issue by tanettrimas** [mozilla/web-ext#1913]
### Is this a feature request or a bug?
Bug

### What is the current behavior?
I am getting security vulnerabilities from yargs-parser as you are using the yargs dependency. Even if I see that you recently updated packages in this PR: https://github.com/mozilla/web-ext/pull/1902, maybe you could try to do this again?

I ran npm audit fix, and got that it needed manual review. 

<img width="717" alt="Skjermbilde 2020-05-23 kl  17 41 46" src="https://user-images.githubusercontent.com/42898343/82734818-ab31c700-9d1d-11ea-8391-516b94215d76.png">

### Version information (for bug reports)

* **Firefox version**: 76.0.1
* **Your OS and version**: Mac OS Catalina 10.15.2
* Paste the output of these commands: npm audit fix

````
node --version 12.14.1 && npm --version 6.14.5 && web-ext --version 4.2.0
````

**Comment by Rob--W** [comment]
This has been fixed in #1902, but we haven't published an update yet.

**Comment by tanettrimas** [comment]
@Rob--W Thanks for the clarification :)

**Comment by rpl** [comment]
Closing as fixed (web-ext v4.3.0 has been just released on npm).
